Utilities/ocpasswordgen/ocpasswordgen.c

```diff
@@ -21,13 +58,34 @@ ENTRY_POINT (
   UINT8   Index;
   UINT8   PasswordHash[SHA512_DIGEST_SIZE];
 
-  DEBUG ((DEBUG_ERROR, "Please enter your password: "));
+  DEBUG ((DEBUG_ERROR, "Please enter your password: \n"));
```

Comment:
In my opinion, better to switch output to printf here and leave it with printf below. DEBUG goes through a lot of intermediate buffers, and may even be disabled. Also, that extra space before `\n` is not needed.

Reply:
Shall we use printf for all userspace tools as well?

Reply:
I believe not, only here it is critical.